lib/src/test/kotlin/network/account/AccountTest.kt

```diff
@@ -334,11 +328,11 @@ class AccountTest {
     @Test
     fun `sign and send declare v3 transaction`() {
         assumeTrue(NetworkConfig.isTestEnabled(requiresGas = true))
-        // Note to future developers experiencing experiencing failures in this test.
+        // Note to future developers experiencing failures in this test.
         // This test sometimes fails due to getNonce receiving higher (pending) nonce than addDeclareTransaction expects
 
         // TODO: (#384) Test v3 transactions on Sepolia
-        assumeTrue(network == Network.GOERLI_INTEGRATION)
+        assumeTrue(network == Network.SEPOLIA_INTEGRATION)
```

Comment:
Does this test pass locally?

If I remember correctly, v3 tests on Sepolia networks should've been added only in #384. If they do work as expected on `SEPOLIA_INTEGRATION` without any additional changes, I would assume `SEPOLIA_TESTNET` tests should work as well and this line could be removed.

Reply:
This also applies to other places where this line is present